### PR TITLE
feat(resources): add a contact page to display members of European Bodies

### PR DESCRIPTION
**Pull request by LeonVreling**

### CODE REVIEW

**Comment by LeonVreling** [comment]
@WikiRik this was something I had been working on quite some time ago, but never got around to fixing the error that was in there.

I would like to make a more pretty page of it in the near future, but I thought it might be nice to already get this online before that with the core functionalities working

**Comment by WikiRik** [comment]
@LeonVreling I'll test this out and merge it if it works. Will quickly check permissions as well and compare it with DPPS

File: src/views/static/Contact.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+  },
+  computed: mapGetters(['services']),
+  mounted () {
+    this.isLoading = true
+    this.fetchData()
+  },
+  methods: {
+    async fetchData () {
+      // TODO: Ask CD what types of bodies they want to have in the address book
```
Comment (WikiRik) [inline src/views/static/Contact.vue]: What does CD want?

**Comment by LeonVreling** [comment]
> ...make a more pretty page of it in the near future...

This will probably be when we are making a fully new frontend

**Comment by LeonVreling** [comment]
Will rework the use of this page later

**Pull request closed without merging**